client/plots/test/oncomatrix.spec.js

```diff
@@ -320,20 +357,56 @@ tape('launch matrix using runproteinpaint with launchGdcMatrix', function (test)
 		},
 		settings: {
 			matrix: {
-				maxGenes: 10,
+				maxGenes: topGeneNum, // asks to load this number of top mutated genes. will be shown in a separate group from termlst
 				maxSample: 10000
 			}
 		},
-		termgroups: [
-			{
-				lst: [
-					{ term: { name: 'IDH1', type: 'geneVariant' } },
-					{ term: { name: 'EGFR', type: 'geneVariant' } },
-					{ id: 'case.disease_type' },
-					{ id: 'case.diagnoses.age_at_diagnosis' }
-				]
+		termgroups: [{ lst: termlst }],
+		opts: {
+			matrix: {
+				callbacks: {
+					'postRender.test': runTests
+				}
+			}
+		}
+	})
+
+	function runTests(matrix) {
+		test.true(
+			holder.querySelectorAll('svg text').length > 200,
+			'should have the expected number of rendered svg text elements'
+		)
+
+		const rowcount = topGeneNum + termlst.length
+		test.equal(
+			select(holder).selectAll('.sjpp-matrix-term-label-g').node().querySelectorAll('text').length,
+			rowcount,
+			`should render ${rowcount} <series> elements`
+		)
+
+		if (test._ok) setTimeout(matrix.destroy, 1000) // TODO why cannot directly run matrix.destroy()
+		test.end()
+	}
+})
+
+tape('top mutated genes from APOLLO-LUAD, CNV only', function (test) {
+	test.timeoutAfter(100000)
+	test.plan(2)
+	const holder = select('body').append('div').node()
+
+	const topGeneNum = 10
+
+	runproteinpaint({
+		holder,
+		noheader: 1,
+		launchGdcMatrix: true,
+		filter0: { op: 'in', content: { field: 'cases.project.project_id', value: ['APOLLO-LUAD'] } },
+		settings: {
+			matrix: {
```

Comment:
since cnv is disabled by default, the view fails and no `sjpp-matrix-term-label-g` is detectable
 @congyu-lu please help turn cnv on in the config here so the map can render

Reply:
just fixed this test